addons/web_editor/static/src/less/web_editor.ui.less

```diff
@@ -284,7 +284,7 @@ html[lang] > body.editor_enable [data-oe-translation-state] {
 
 // MANIPULATORS
 #oe_manipulators {
-    @odoo-handle-edge-size: 8px;
+    @odoo-handle-edge-size: 1px;
```

Comment:
So... I removed the 'blinking effect' each time you select a snippet but I had to decrease again the handler's size ( like it was before).
Since handlers now have bg color, they were visually overlapping each other causing weird results :/

I don't think that it's a big issue though, you'll have to be more careful dragging the button if the margin is 0. In all other cases the overall experience it's much more improved, you can even drag a random point in the margin area (if a minimum margin is applied).
Let me know, thanks!